sdk/core/azure-core-management/src/main/java/com/azure/core/management/implementation/polling/PollOperation.java

```diff
@@ -50,12 +50,12 @@ public final class PollOperation {
         return pollingContext -> {
             PollingState pollingState = PollingState.from(serializerAdapter, pollingContext);
             if (pollingState.getOperationStatus().isComplete()) {
-                return pollResponseMonoFromPollingState(serializerAdapter, pollResultType, pollingState);
+                return Mono.just(pollResponseFromPollingState(serializerAdapter, pollResultType, pollingState));
```

Comment:
Use `Mono.defer()` instead of `Mono.just()`.

Reply:
done